templates/jobservice/jobservice-secrets.yaml

```diff
@@ -1,10 +1,22 @@
+{{- $secret := include "harbor.jobservice" . }}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: "{{ template "harbor.jobservice" . }}"
+  name: {{ $secret }}
   labels:
 {{ include "harbor.labels" . | indent 4 }}
 type: Opaque
 data:
+  {{- if .Release.IsUpgrade }}
+    {{- $prevSecret := (lookup "v1" "Secret" .Release.Namespace $secret) }}
+    {{- if .Values.jobservice.secret }}
+    JOBSERVICE_SECRET: {{ .Values.jobservice.secret | b64enc | quote }}
```

Comment:
The indentation here is incorrect. Only detectable during `helm upgrade`.

Reply:
Yeah, I did it on purpose; does it break anything?